### PR TITLE
LIMS-649: Increase max beam size for i23 in run stats page

### PR DESCRIPTION
**Pull request by ndg63276**
Ticket: [LIMS-649](https://jira.diamond.ac.uk/browse/LIMS-649)

* I23 use bigger beam sizes than other MX beamlines
* Use larger max value for x axis on beam size x/y histograms when beamline is i23, or comparing all beamlines

### CODE REVIEW

File: api/src/Page/Vstat.php [hunk 1]
```diff
@@ -944,10 +944,15 @@ function _parameter_histogram()
         $beamlines = $this->_get_beamlines_from_type($this->ty);
         $bls = implode('\', \'', $beamlines);
 
+        $max_beam_size = 600;
```
Comment (MattPrit) [inline api/src/Page/Vstat.php:947]: Is there any particular reason for having the i23 case be the default?

File: api/src/Page/Vstat.php [hunk 2]
```diff
@@ -944,10 +944,15 @@ function _parameter_histogram()
...
+        if ($this->has_arg('bl') && $this->arg('bl') != 'i23') {
```
Comment (MattPrit) [inline api/src/Page/Vstat.php:948]: Though it probably won't be needed here, it may be better to use `!==`

File: api/src/Page/Vstat.php (hunk 1 above)
Comment (ndg63276) [reply api/src/Page/Vstat.php:947]: It was so that the "Compare all beamlines" tickbox showed the larger beamsizes.

**Comment by ndg63276** [comment]
I removed the hard coded start and end values, and the references to i23, and now just use whatever data comes out.

File: api/src/Page/Vstat.php [hunk 3]
```diff
@@ -987,9 +987,15 @@ function _parameter_histogram()
...
+        $min = null;
+        $max = null;
         $bls = array();
-        foreach ($hist as $h)
+        foreach ($hist as $h) {
             $bls[$h['BEAMLINENAME']] = 1;
+            if (is_null($max) || $h['X'] > $max) $max = $h['X'];
+            if (is_null($min) || $h['X'] < $min) $min = $h['X'];
+        }
+        $min = is_null($min) ? 0 : floor($min/$binSize) * $binSize; // make min align with bin size, or 0 for no data
```
Comment (ndg63276) [inline api/src/Page/Vstat.php]:
```suggestion
        $min = is_null($min) ? 0 : intval(floor($min/$binSize) * $binSize); // make min align with bin size, or 0 for no data
```

**Review by John-Holt-Tessella — APPROVED** [review]

**Pull request merged**